### PR TITLE
add dataZoom for graph

### PR DESCRIPTION
**Pull request by lvybriage**
1. default metric graph: 支持鼠标滚轴缩放扩大效果，Y轴变化
2. Trial duration: 支持鼠标滚轴缩放扩大效果，Y轴变化，X轴保持不变
3. Intermediate Result: 支持鼠标滚轴缩放扩大效果，Y轴变化，X轴保持不变
4. Hyper-parameter graph: To Do. I have a issue in echarts, will support in other PR.
5. hyper-parameter graph default metric轴数值实时计算
6. hyper-parameter graph yAxis values should be not bold. [Echarts Issue](https://github.com/apache/incubator-echarts/issues/11685)
image

image

Please help review this PR.

### CODE REVIEW

**Comment by ultmaster** [comment]
It's more useful to have y-zoom instead of x-zoom in "Intermediate result", if supporting both is difficult.

As for "default metric" and "trial duration", why bother?

File: src/webui/package.json [hunk 1]
```diff
@@ -12,7 +12,7 @@
     "copy-to-clipboard": "^3.0.8",
     "css-loader": "0.28.7",
     "dotenv": "^8.0.0",
-    "echarts": "^4.1.0",
+    "echarts": "^4.5.0",
```
Comment (liuzhe-lz) [inline src/webui/package.json]: You need to commit `yarn.lock`.

Comment (lvybriage) [reply src/webui/package.json]: OK. Thanks.

File: src/webui/yarn.lock [hunk 2]
```diff
@@ -5,34 +5,29 @@
 "@ant-design/icons-react@~1.1.2":
   version "1.1.5"
   resolved "https://registry.yarnpkg.com/@ant-design/icons-react/-/icons-react-1.1.5.tgz#1b03da8dcced2a4bb982ef7b25c1d24014c35a68"
-  integrity sha512-p2ybKfO/r2lC1RZu4rDY2VBDZq2zqAaJzf/B1HrKTxGo8/mM1zOOEoob/LRXZphJ9jD5wCcTdcmcB9YMaAWW4Q==
```
Comment (liuzhe-lz) [inline src/webui/yarn.lock]: The PR is okay but I suggest you to upgrade your yarn version.

**Pull request merged**